custom/reach/const.py

```diff
@@ -0,0 +1,25 @@
+from __future__ import absolute_import
+from __future__ import unicode_literals
+
+from django.utils.translation import ugettext_lazy as _
+
+INDICATOR_LIST = {
+    'registered_eligible_couples': _('Registered Eligible Couples'),
+    'registered_pregnancies': _('Registered Pregnancies'),
+    'registered_children': _('Registered Children'),
+    'couples_family_planning': _('Couples using Family Planning Method'),
+    'high_risk_pregnancies': _('High Risk Pregnancies'),
+    'institutional_deliveries': _('Institutional Deliveries'),
+}
+
+NUMERIC = 'numeric'
+PERCENT = 'percent'
+
+COLORS = {
+    'violet': '#8A2BE2',
+    'blue': '#00BFFF',
+    'mediumblue': '#0000CD',
+    'aqua': '#05EDFF',
+    'orange': '#FFA500',
+    'darkorange': '#FF8C00',
+}
```

Comment:
Can you update with the following codes for the correct reports:

Registered Eligible Couples : #725CA4
Registered Pregnancies : #04AEE6
Registered Children : #F5BB5D
Couples using Family Planning Method : #1CC6CC
High Risk Pregnancies : #FF8300
Institutional Deliveries : #004EBC